tests/test_cli.py

```diff
@@ -136,6 +136,8 @@ def test_browser_argument(browser_arg, capsys, platform):
     """Test arguments for the browser option."""
     for browser_opt in VALID_CMD_OPTS[2]:
         try:
+            if get_browser(browser_arg) is None:
+                raise AssertionError("browser is unavailable")
```

Comment:
What's this for?

Reply:
`VALID_BROWSER_ARG` may contain browsers not supported on current platform. But we just fixed the unhandled `AssertionError` so this means a browser in `VALID_BROWSER_ARG` and not supported on current platform will hit a `SystemExit` error instead of the `AssertionError` causing the test to fail. So this check along with another one beneath it bypass the SyatemExit and throws an AssertionError instead so it can be handled like the rest of them. Do you have an alterntive way to handle it?

Reply:
Ah okay. That seems like a good enough way to handle it. Maybe document it with a small comment here?

Reply:
@Samyak2 it is done now.